subprojects/logging/src/main/java/org/gradle/internal/logging/events/RenderableOutputEvent.java

```diff
@@ -16,13 +16,16 @@
 
 package org.gradle.internal.logging.events;
 
+import org.gradle.api.Nullable;
 import org.gradle.api.logging.LogLevel;
 import org.gradle.internal.logging.text.StyledTextOutput;
 
 public abstract class RenderableOutputEvent extends CategorisedOutputEvent {
+    private final OperationIdentifier operationId;
```

Comment:
We wouldn't add any more information to the id. The operation type would be an additional property of the progress start event.